plugins/hybrid_analysis/plugin.spec.yaml

```diff
@@ -1,145 +1,1109 @@
+      title: Compromised Hosts
       type: '[]string'
+      description: Compromised hosts
       required: false
     hosts:
+      title: Hosts
+      type: '[]string'
       description: Hosts
+      required: false
+    total_network_connections:
+      title: Total Network Connections
+      type: integer
+      description: Total network connections
+      required: false
+    total_processes:
+      title: Total Processes
+      type: integer
+      description: Total processes
+      required: false
+    total_signatures:
+      title: Total Signatures
+      type: integer
+      description: Total signatures
+      required: false
+    extracted_files:
+      title: Extracted Files
+      type: '[]extracted_files'
+      description: Extracted files
+      required: false
+    file_metadata:
+      title: File Metadata
+      type: file_metadata
+      description: File metadata
+      required: false
+    processes:
+      title: Processes
+      type: '[]processes'
+      description: Processes
+      required: false
+    tags:
+      title: Tags
       type: '[]string'
+      description: Tags
+      required: false
+    mitre_attcks:
+      title: MITRE Attcks
+      type: '[]mitre_attcks'
+      description: MITRE attcks
+      required: false
+    submissions:
+      title: Submissions
+      type: '[]submissions'
+      description: Submissions
+      required: false
+    network_mode:
+      title: Network Mode
+      type: string
+      description: Network mode
+      required: false
+    machine_learning_models:
+      title: Machine Learning Models
+      type: '[]machine_learning_models'
+      description: Machine learning models
+      required: false
+  search_term:
+    id:
+      title: ID
+      description: Name of search term which was used
+      type: string
+      required: false
+    value:
+      title: Value
+      description: Value of search term
+      type: string
+      required: false
+  result:
+    verdict:
+      title: Verdict
+      description: File verdict e.g. malicious
+      type: string
+      required: false
+    av_detect:
+      title: AV detect
+      description: AV MultiScan Detection Percentage
+      type: string
+      required: false
+    threat_score:
+      title: Thread score
+      description: Confidence value of VxStream Sandbox in the verdict; lies between
+        0 and 100
+      type: integer
+      required: false
+    vx_family:
+      title: VX Family
+      description: VX Family e.g. Trojan.Generic
+      type: string
+      required: false
+    job_id:
+      title: Job ID
+      description: Job ID when file was submited
+      type: string
+      required: false
+    sha256:
+      title: SHA256
+      description: SHA256 hash
+      type: string
+      required: false
+    environment_id:
+      title: Environment ID
+      description: The environment that was used for analysis
+      type: integer
+      required: false
+    analysis_start_time:
```

Comment:
If this is input, can you please change it to Date, so that user is able to pick the date.

Reply:
This type is used in output, sou user is not selecting anything from GUI